.github/workflows/merge.yml

```diff
@@ -16,6 +16,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     name: "Merge"
+    env:
+      RELEASE_VERSION: 2.10-SNAPSHOT
```

Comment:
I'm thinking at some point we might be able to extract it from the pom (like we do with the egeria-react-ui build for npm) so it's in a single place, but this is fine for now.